src/main/java/com/ibm/crypto/plus/provider/EdDSAPrivateKeyImpl.java

```diff
@@ -118,7 +118,7 @@ public EdDSAPrivateKeyImpl(OpenJCEPlusProvider provider, byte[] encoded)
             // to fit with GSKit and sets params
 
             checkLength(this.curve);
-            int priv_size = CurveUtil.getPrivateCurveSize(curve);
+            int priv_size = CurveUtil.getDEREncodingSize(curve);
```

Comment:
Should we change the name of the variable to `encodingSize`?

Reply:
Yes.